lib/tasks/import.rake

```diff
@@ -21,18 +21,14 @@ namespace :import do
     organisation_count = 0
     child_organisation_count = 0
     relationships = {}
-    host = ENV.fetch('LEGACY_HOST')
+    host = 'https://data.gov.uk/'
     path = 'data/dumps/data.gov.uk-ckan-meta-data-latest.organizations.jsonl.zip'
     url = URI::join(host, path).to_s
+    file = download_data('latest_legacy_organisations', url, logger)
 
     logger.info 'Processing parent organisations'
-
-    file = Tempfile.new('latest_organisations')
-    file.binmode
-    file.write(RestClient.get(url).body)
-    file.close
-
-    read_json_from_zip(file) do |obj|
+    read_json_from_zip(file, logger) do |obj|
+      # TODO - move to MetadataTools.rb
```

Comment:
There is still a fair amount of work that needs to be done to refactor this file further, but not a high priority right now